test/burials/config/burialAllowance.unit.spec.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, submitForm, getFormDOM } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/burials/config/form.js';
+
+describe('Burials burial allowance', () => {
+  const { schema, uiSchema, depends } = formConfig.chapters.benefitsSelection.pages.burialAllowance;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input, select, textarea').length).to.equal(5);
+  });
+
+  it('should show errors when required fields are empty', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+    submitForm(form);
+    expect(Array.from(formDOM.querySelectorAll('.usa-input-error')).length).to.equal(1);
```

Comment:
In the next test, you get the length of a `.querySelectorAll()` without doing `Array.from()`. Is it necessary?